<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>1. Choose file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>2. Choose language</string>
    </property>
   </widget>
   <widget class="QListWidget" name="languageSelect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>141</width>
      <height>281</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Chinese (Simplfied)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chinese (Traditional)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Czech</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Danish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dutch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Finnish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>French</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>German</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Greek</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hungarian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Italian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Japanese</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Korean</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Norwegian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Polish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Portuguese</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spanish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Swedish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Turkish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Russian</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_api">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>3. Enter API Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Get it: https://ocr.space/OCRAPI</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="apiEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="apiKeySubmit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>4. Generate text!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateOCRButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="displayText">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="exportToFileButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Export to file</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>632</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>

